vendor-extra/JatsContentBundle/tests/ViewConverter/ArticleTitleHeadingVisitorTest.php

```diff
@@ -93,7 +93,7 @@ public function it_sets_the_text_argument() : void
         $xml = FluentDOM::load(
             <<<XML
 <jats:article-title xmlns:jats="http://jats.nlm.nih.gov">
-    foo <jats:italic>bar</jats:italic> baz
+    foo <jats:italic>bar</jats:italic> baz <jats:bold>bold</jats:bold>
```

Comment:
I'd revert this, we only need to trigger multiple nodes here, so don't need to test for all the possible elements.